recipes/stglib/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "0.2.0" %}
+
+package:
+  name: stglib
+  version: {{ version }}
+
+source:
+  url: https://github.com/dnowacki-usgs/stglib/archive/v{{ version }}.tar.gz
+  sha256: 0c2860cde4bfda74e48cffb55ff582a684b895192e6c961b47976af467fe37f6
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python
+    - matplotlib
```

Comment:
Can this be `matplotlib-base` instead? `matplotlib` provides the Qt backend, but if you don't need the interactive images and just need to render them, then you should use `matplotlib-base`.